converter/datamodel.py

```diff
@@ -435,7 +487,7 @@ def from_magetab(cls, study_info):
         experiment_type = comments.get("experiment_type", [])
 
         return cls(alias, accession, title, description, protocolrefs, projectref,
-                   ef_objects, ed_objects, experiment_type, date_of_experiment,
+                   ef_objects, ed_objects, experiment_type, date_of_experiment, None,
```

Comment:
maybe be explicit here with the argument as you have been in all other instances of cls creation?